### PR TITLE
feat: (KW-149) Add CICO to vendor object

### PR DESCRIPTION
**Pull request by Nashmod**
### Description

This pr was too add the CICO boolean to the vendor object coming from the Vendors API.

### Other changes

_Describe any minor or "drive-by" changes here._

### Tested

_An explanation of how the changes were tested or an explanation as to why they don't need to be._

### How others should test

Build the app and check the metro log to see it the set_vendors saga contains the cico key.

### Related issues

- Fixes #[issue number here]

### Backwards compatibility

_Brief explanation of why these changes are/are not backwards compatible._

### CODE REVIEW

File: src/vendors/types.ts [hunk 1]
```diff
@@ -10,6 +10,7 @@ export type Vendor = {
   currencies: Array<string>
   address?: string
   phoneNumber?: string
+  cico?: boolean
```
Comment (shottah) [inline src/vendors/types.ts]: Could you split this into 'acceptsGuilder: boolean, providesGuilder: boolean'?

**Review by shottah — CHANGES_REQUESTED** [review]

File: src/vendors/types.ts (hunk 1 above)
Comment (Nashmod) [reply src/vendors/types.ts]: Done!

**Review by Nashmod — COMMENTED** [review]
fixed

**Review by shottah — CHANGES_REQUESTED** [review]
This PR isn't showing a UI element on the `VendorDetail` based on whether the vendor Accepts cGuilder (see Figma designs).

**Review by shottah — APPROVED** [review]

**Pull request merged**